F50/Preferences/Preferences.android.cs

```diff
@@ -1,14 +1,130 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
+using System.Globalization;
+using Android.App;
+using Android.Content;
+using Android.Preferences;
 
 namespace Xamarin.F50
 {
 	public partial class Preferences
```

Comment:
Do we need locks at all?

Reply:
Yeah. I wanted to evaluate if we truly did or not first. On android it seems in theory no but I'm seeing some posts that seem to have problems in practice so better safe than sorry here I think.